todolist.go

```diff
@@ -0,0 +1,447 @@
+package main
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+
+	_ "modernc.org/sqlite"
+)
+
+func main() {
+	checkTable()
+	port := os.Getenv("TODO_PORT")
+	if len(port) == 0 {
+		port = "7540"
+	}
+	serv := fmt.Sprintf("0.0.0.0:%s", port)
+	http.Handle("/", http.FileServer(http.Dir("./web")))
+	http.HandleFunc("/api/nextdate", apiNextDateHandle)
```

Comment:
Тут лучше писать так
```suggestion
	http.HandleFunc("GET /api/nextdate", apiNextDateHandle)
```
Тогда к этой ручке нельзя будет обратиться с другим методом (а проверки внутри у тебя нет)

Так же для `/tasks` можно сделать отдельные функции и тут через роутер их уже распределять

Reply:
Исправил и сделал для /task отдельные функции